script.js

```diff
@@ -0,0 +1,27 @@
+// A function that adds and remove the class "active" on the section you click on.
+// We haven't really talked about ´this´ yet, but we will... ;)
+// console.log(this) to see how it works
+
+// Selects an HTML element, and calls a function which will be executed when the element is clicked.
+const accordionHeader1 = document.getElementById('accordion-header1');
+const accordionHeader2 = document.getElementById('accordion-header2');
+const accordionHeader3 = document.getElementById('accordion-header3');
+
+
+accordionHeader1.onclick = () => {
+  const accordionTextContainer1 = accordionHeader1.nextElementSibling;
+  accordionTextContainer1.classList.toggle('active');
+
+
+}
+accordionHeader2.onclick = () => {
+  const accordionTextContainer2 = accordionHeader2.nextElementSibling;
+  accordionTextContainer2.classList.toggle('active');
+  accordionTextContainer2.classList.toggle('color-dark');
+
+}
+accordionHeader3.onclick = () => {
+  const accordionTextContainer3 = accordionHeader3.nextElementSibling;
+  accordionTextContainer3.classList.toggle('active');
+
+}
```

Comment:
Thought practice, what if the code said:
```Javascript
accordionHeader1.onclick = () => {
  const textContainer = accordionHeader1.nextElementSibling;
  textContainer .classList.toggle('active');


}
accordionHeader2.onclick = () => {
  const textContainer = accordionHeader2.nextElementSibling;
  textContainer .classList.toggle('active');
  textContainer .classList.toggle('color-dark');

}
accordionHeader3.onclick = () => {
  const textContainer = accordionHeader3.nextElementSibling;
  textContainer .classList.toggle('active');

}
```

How would this change the functionality.  We will approach this subject soon.  If you are interested in understanding more.  Look into "Javascript scope"